project_variables.go

```diff
@@ -173,6 +175,7 @@ type UpdateProjectVariableOptions struct {
 	EnvironmentScope *string            `url:"environment_scope,omitempty" json:"environment_scope,omitempty"`
 	Filter           *VariableFilter    `url:"filter,omitempty" json:"filter,omitempty"`
 	Masked           *bool              `url:"masked,omitempty" json:"masked,omitempty"`
+	Hidden           *bool              `url:"hidden,omitempty" json:"hidden,omitempty"`
```

Comment:
From the [docs](https://docs.gitlab.com/ee/api/project_level_variables.html#update-a-variable) I don't see that this field is supported in the update API. Did `hidden` work for you?

Reply:
This feature is not matured, You can close the MR or leave it open until this field is available in update api.

Reply:
I think it's up to you. We can already merge what is supported and iterate. No need to wait for a "matured" upstream API. 

IMHO, we should move on with what is supported upstream.

Reply:
Resolving this as I have removed the field from update options since GitLab doesn't support it